joomla/joomla-cms
# [5.1] CLI: Renaming and improving output of core:update:check

### Summary of Changes
The commands for core updates are not consistent right now. This is a first attempt at getting a system into these commands and improving the output. This PR renames the `core:check-updates` command to `core:update:check` (and adds an alias to keep it backwards compatible) and extends the output to display the current Joomla version and the update channel we are currently on.

An additional command `core:update:set-channel` will allow to set the update channel in the future.


### Testing Instructions
1. call `php cli/joomla.php`
2. call `php cli/joomla.php core:update:check`
3. call `php cli/joomla.php core:check-updates`

### Actual result BEFORE applying this Pull Request
to 1.: See the command listed as `core:check-updates`
to 2.: See the command fail
to 3.: See the title "Joomla updates" and the text "You already have the latest Joomla version..."


### Expected result AFTER applying this Pull Request
to 1.: See the command listed as `core:update:check` and with an alias of `core:check-updates`
to 2.: See the command go through successfully with the title "Joomla Update Status" and the currently selected update channel and the  current Joomla version in addition to the next available version.
to 3.: See the command to also work successfully, like in 2.


### Link to documentations
Please select:
